packages/x-adapter/src/index.ts

```diff
@@ -1,6 +1,5 @@
-export * from './endpoint-adapter.factory';
+export * from './endpoint-adatpter';
```

Comment:
Fix typo in folder name

```suggestion
export * from './endpoint-adapter';
```

Reply:
done